### PR TITLE
DOC-2363: Included `itemprop`, `itemscope`, `itemtype` as valid HTML5 attributes in the core schema.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2363

Site: [Staging branch](http://docs-feature-71-doc-2363tiny-9932.staging.tiny.cloud/docs/tinymce/latest/7.1-release-notes/#included-itemprop-itemscope-itemtype-as-valid-html5-attributes-in-the-core-schema)

Changes:
* Added improvement documentation for: Included `itemprop`, `itemscope`, `itemtype` as valid HTML5 attributes in the core schema.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by ArvinJ-H — APPROVED** [review]
LGTM

File: modules/ROOT/pages/7.1-release-notes.adoc [hunk 1]
```diff
@@ -114,6 +114,25 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 // CCFR here.
 
+=== Included itemprop, itemscope, itemtype as valid HTML5 attributes in the core schema.
+// #TINY-9932
+
+Previously, using `itemprop`, `itemscope` and `itemtype` as valid HTML5 attributes within the editor was not supported as they were not included in the update in v6.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.1-release-notes.adoc:120]: What do we mean by v6 here?

File: modules/ROOT/pages/7.1-release-notes.adoc [hunk 2]
```diff
@@ -114,6 +114,25 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
...
+
+As a consequence, users faced difficulties in effectively working with content that used these attributes resulting in compatibility issues or data misrepresentation.
+
+{productname} {product-version} addresses this issue, now, the missed attributes have been added to the HTML5 schema, ensuring that users can now use content containing these attributes such as;
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.1-release-notes.adoc:124]:
```suggestion
{productname} {release-version} addresses this issue, now, the missed attributes have been added to the HTML5 schema, ensuring that users can now use content containing these attributes such as;
```

**Review by FarzadHayat — APPROVED** [review]

File: modules/ROOT/pages/7.1-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/7.1-release-notes.adoc:136]:
```suggestion
Previously, using `itemprop`, `itemscope` and `itemtype` as valid HTML5 attributes within the editor was not supported.
```

**Pull request merged**